tutor/test_cc1_08_StudentWorkAssignments.py

```diff
@@ -0,0 +1,214 @@
+"""Concept Coach v1, Epic 08 - StudentsWorkAssignments."""
+
+import json
+import os
+import pytest
+import unittest
+
+from pastasauce import PastaSauce, PastaDecorator
+from random import randint
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expect
+from staxing.assignment import Assignment
+from staxing.helper import Teacher, Student
+
+basic_test_env = json.dumps([{
+    'platform': 'OS X 10.11',
+    'browserName': 'chrome',
+    'version': '48.0',
+    'screenResolution': "1024x768",
+}])
+BROWSERS = json.loads(os.getenv('BROWSERS', basic_test_env))
+TESTS = os.getenv(
+    'CASELIST',
+    str([7691, 7692, 7693, 7694, 7695,
+         7696, 7697, 7698, 7699, 7700,
+         7701, 7702, 7703])
+)
+
+
+@PastaDecorator.on_platforms(BROWSERS)
+class TestStudentsWorkAssignments(unittest.TestCase):
+    """CC1.08 - Students Work Assignments."""
+
+    def setUp(self):
+        """Pretest settings."""
+        self.ps = PastaSauce()
+        teacher = Teacher(username='teacher100', password='password',
+                          site='https://tutor-qa.openstax.org/')
+        teacher.login()
+        courses = teacher.find_all(By.CLASS_NAME,
+                                   'tutor-booksplash-course-item')
+        assert(courses), 'No courses found.'
+        if not isinstance(courses, list):
+            courses = [courses]
+        course_id = randint(0, len(courses) - 1)
+        self.course = courses[course_id].get_attribute('data-title')
+        teacher.select_course(title=self.course)
+        teacher.goto_course_roster()
+        section = '%s' % randint(100, 999)
+        try:
+            wait = teacher.wait_time
+            teacher.change_wait_time(3)
+            teacher.find(By.CLASS_NAME, '-no-periods-text')
```

Comment:
Requiring the test system to match the current code base of the system is cumbersome. There are two tracts right now: Bernard is working on the JS tests (although I need to pester him more to push his changes and make PRs) and our current intern group which will be working on test case breakdowns and Python tests.

The test case above is a PoC for QA; it's not meant to be structurally sound in the long run, just semi-functional right now for the test flow:
TestRail => Jenkins => GitHub => Jenkins => SauceLabs => Jenkins => TestRail